bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx

```diff
@@ -108,7 +109,24 @@ const VideoListItem = (props) => {
       talking={talking}
       animations={animations}
     >
-      <Styled.LoadingText>{name}</Styled.LoadingText>
+      <UserAvatarVideo
+        user={user}
+        voiceUser={voiceUser}
+      />
+      <Styled.BottomBar>
```

Comment:
Since the actions bar/mic status is now being rendered, shouldn't it fall under the same UI `squeezing` logic introduced in https://github.com/bigbluebutton/bigbluebutton/pull/14732?

Reply:
Yeah, it should. Fixed in https://github.com/bigbluebutton/bigbluebutton/pull/14938/commits/570d341e4657952c9b0a20af0cc1347b2a70bb0e